### DISCUSSION TITLE
How to use Simplicity Commander?

### DISCUSSION THREAD

**Discussion by miroslavpetrov** [SiliconLabs/arduino#53] [Q&A] [answered]
@silabs-bozont I saw that in few issues you suggest that the users can erase the flash of the EFR32 using the Simplicity Commander which comes with the Silabs Arduino core. On MacOS the commander doesn't seem to be able to find the Nano Matter board connected to it.
![Screenshot 2024-05-16 at 22 10 53](https://github.com/SiliconLabs/arduino/assets/25277610/ecf158a3-8ef8-4d68-be19-0282d1a29241)

I have Jlink Utilities installed and JlinkConfing can find the device. 
![Screenshot 2024-05-16 at 22 13 00](https://github.com/SiliconLabs/arduino/assets/25277610/cc246901-48b8-43b9-bb51-43b8c8cb8001)

I also tried to run commander from cli and to set the adapter using adapter probe --serialno but it doesn't work again.
![Screenshot 2024-05-16 at 22 14 46](https://github.com/SiliconLabs/arduino/assets/25277610/72d5384c-c562-4d0f-b46c-9b3c1faaf37c)

Can you share with me how can I use the commander utility with Arduino Nano Matter?

**Comment by silabs-bozont** [comment, marked as answer]
Hello @miroslavpetrov,
As opposed to the Silicon Labs branded kits the Arduino Nano Matter does not use J-Link - therefore it's not compatible with Simplicity Commander or any of the J-Link utilities.
The Nano Matter uses OpenOCD - which is a bit different. The Arduino IDE now offers erasing support as well with the 2.0.0 version - so you can also erase your board just by using the Arduino IDE.
To do that - select `Tools > Burn Bootloader` after selecting `OpenOCD` as your programmer in the Arduino IDE. This will do a full erase on your board, then burn the bootloader.

**Reply by miroslavpetrov** [reply]
thank you for the clarification.
